phpstan.neon.dist

```diff
@@ -117,3 +117,5 @@ parameters:
 		-
 			message: '#Call to method get(Class|Headers|StatusCode)\(\) on an unknown class Symfony\\Component\\ErrorRenderer\\Exception\\FlattenException\.#'
 			path: %currentWorkingDirectory%/src/Action/ExceptionAction.php
+
+		- '"#Symfony\\Component\\HttpKernel\\EventListener\\ErrorListener#'
```

Comment:
```suggestion
		- '#Symfony\\Component\\HttpKernel\\EventListener\\ErrorListener#'
```

my bad

Reply:
I think this should be:

1. moved to the bottom-most "forward compatibility" section
2. match the entire phpstan error message
3. scoped to the relevant path(s)